### PR TITLE
feat: add placeholders to form input fields (#89) 

### PR DESCRIPTION
**Pull request by Sarlin-7757**
feat: add placeholders to form input fields (#89)

Fix issue mentioned in #89 we need placeholders for input boxes everywhere. 

This commit addresses the issue mentioned in #89 by implementing the necessary changes to resolve it.

### CODE REVIEW

**Comment by Sarlin-7757** [comment]
Kindly review this pull request. @bhanuvrat  @bind-ritu1211

File: lib/vachan_web/live/list_live/form_component.ex [hunk 1]
```diff
@@ -19,7 +19,7 @@ defmodule VachanWeb.ListLive.FormComponent do
         phx-change="validate"
         phx-submit="save"
       >
-        <.input field={@form[:name]} type="text" label="Name" />
+        <.input field={@form[:name]} type="text" label="Name" placeholder="Enter list name" />
```
Comment (bind-ritu1211) [inline lib/vachan_web/live/list_live/form_component.ex:22]: update placeholder text  >> placeholder="List Name"

File: lib/vachan_web/live/person_live/form_component.ex [hunk 2]
```diff
@@ -19,9 +19,9 @@ defmodule VachanWeb.PersonLive.FormComponent do
         phx-change="validate"
         phx-submit="save"
       >
-        <.input field={@form[:first_name]} type="text" label="First name" />
-        <.input field={@form[:last_name]} type="text" label="Last name" />
-        <.input field={@form[:email]} type="email" label="Email" />
+        <.input field={@form[:first_name]} type="text" label="First name" placeholder="John"/>
```
Comment (bind-ritu1211) [inline lib/vachan_web/live/person_live/form_component.ex:22]: update placeholder text  >> placeholder="First Name"

File: lib/vachan_web/live/person_live/form_component.ex [hunk 3]
```diff
@@ -19,9 +19,9 @@ defmodule VachanWeb.PersonLive.FormComponent do
...
+        <.input field={@form[:last_name]} type="text" label="Last name" placeholder="Doe"/>
```
Comment (bind-ritu1211) [inline lib/vachan_web/live/person_live/form_component.ex:23]: update placeholder text  >> placeholder="Last Name"

File: lib/vachan_web/live/person_live/form_component.ex [hunk 4]
```diff
@@ -19,9 +19,9 @@ defmodule VachanWeb.PersonLive.FormComponent do
...
+        <.input field={@form[:email]} type="email" label="Email" placeholder="john.doe@example.com"/>
```
Comment (bind-ritu1211) [inline lib/vachan_web/live/person_live/form_component.ex:24]: update placeholder text  >> placeholder="E-mail"

File: lib/vachan_web/live/sender_profile_live/form_component.ex [hunk 5]
```diff
@@ -19,14 +19,14 @@ defmodule VachanWeb.SenderProfileLive.FormComponent do
...
       >
-        <.input field={@form[:title]} type="text" label="Sender Profile's title" />
-        <.input field={@form[:name]} type="text" label="Sender's Name" />
-        <.input field={@form[:email]} type="email" label="Sender's Email" />
-
-        <.input field={@form[:smtp_host]} type="text" label="SMTP Server's Address" />
-        <.input field={@form[:smtp_port]} type="number" label="SMTP Server's Port" />
-        <.input field={@form[:username]} type="text" label="SMTP username" />
-        <.input field={@form[:password]} type="text" label="SMTP password" />
+        <.input field={@form[:title]} type="text" label="Sender Profile's title" placeholder="Marketing Campaign"/>
```
Comment (bind-ritu1211) [inline lib/vachan_web/live/sender_profile_live/form_component.ex:22]: update placeholder text  >> placeholder="Profile's title"

File: lib/vachan_web/live/sender_profile_live/form_component.ex [hunk 6]
```diff
@@ -19,14 +19,14 @@ defmodule VachanWeb.SenderProfileLive.FormComponent do
...
+        <.input field={@form[:name]} type="text" label="Sender's Name" placeholder="John Doe"/>
```
Comment (bind-ritu1211) [inline lib/vachan_web/live/sender_profile_live/form_component.ex:23]: update placeholder text  >> placeholder="Name"

File: lib/vachan_web/live/sender_profile_live/form_component.ex [hunk 7]
```diff
@@ -19,14 +19,14 @@ defmodule VachanWeb.SenderProfileLive.FormComponent do
...
+        <.input field={@form[:email]} type="email" label="Sender's Email" placeholder="john.doe@example.com" />
```
Comment (bind-ritu1211) [inline lib/vachan_web/live/sender_profile_live/form_component.ex:24]: update placeholder text  >> placeholder="E-mail"

File: lib/vachan_web/live/sender_profile_live/form_component.ex [hunk 8]
```diff
@@ -19,14 +19,14 @@ defmodule VachanWeb.SenderProfileLive.FormComponent do
...
+
+        <.input field={@form[:smtp_host]} type="text" label="SMTP Server's Address" placeholder="smtp.example.com" />
```
Comment (bind-ritu1211) [inline lib/vachan_web/live/sender_profile_live/form_component.ex:26]: update placeholder text  >> placeholder="Server's address"

File: lib/vachan_web/live/sender_profile_live/form_component.ex [hunk 9]
```diff
@@ -19,14 +19,14 @@ defmodule VachanWeb.SenderProfileLive.FormComponent do
...
+        <.input field={@form[:smtp_port]} type="number" label="SMTP Server's Port" placeholder="587" />
```
Comment (bind-ritu1211) [inline lib/vachan_web/live/sender_profile_live/form_component.ex:27]: update placeholder text  >> placeholder="Server's port"

File: lib/vachan_web/live/sender_profile_live/form_component.ex [hunk 10]
```diff
@@ -19,14 +19,14 @@ defmodule VachanWeb.SenderProfileLive.FormComponent do
...
+        <.input field={@form[:username]} type="text" label="SMTP username"  placeholder="smtp_user"/>
```
Comment (bind-ritu1211) [inline lib/vachan_web/live/sender_profile_live/form_component.ex:28]: update placeholder text  >> placeholder="Username"

File: lib/vachan_web/live/sender_profile_live/form_component.ex [hunk 11]
```diff
@@ -19,14 +19,14 @@ defmodule VachanWeb.SenderProfileLive.FormComponent do
...
+        <.input field={@form[:password]} type="text" label="SMTP password" placeholder="*********"/>
```
Comment (bind-ritu1211) [inline lib/vachan_web/live/sender_profile_live/form_component.ex:29]: update placeholder text  >> placeholder="Password...."

**Comment by Sarlin-7757** [comment]
Okk will do

**Review by bind-ritu1211 — APPROVED** [review]

**Comment by bind-ritu1211** [comment]
@Sarlin-7757  Please resolve conflits.

**Pull request closed without merging**